src/basics/arrays/intersection/index.js

```diff
@@ -4,6 +4,17 @@
  * @returns {Array<Number>} The intersection of the two lists
  * Bonus: try to implement this without a `for` or `while` loop
  */
-function intersection(listA, listB) {}
+function intersection(listA, listB) {
+    arrayNumb = [];
+    for (let i = 0; i < listA.length; i++) {
+        for (let j = 0; j < listB.length; j++) {
+            if (listA[i]==listB[j]) {
+               arrayNumb.push(listA[i]) 
+            }   
+        }
+        
+    }
```

Comment:
Hint about implementation with loops: try to use the `filter` and `includes` methods of Array.